pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -70,5 +75,135 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			Expect(err).To(HaveOccurred())
 
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// sa can create project
+		ginkgo.It("sa can create project", func() {
+
+			h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+
+			_, err := createNamespace("cust-ns", h)
```

Comment:
IIRC, they shouldn't be able to create namespaces, but projects. They're considered different in the api. Might want to double check this

Reply:
+1 changed to create projectRequest object